project/Dependencies.scala

```diff
@@ -19,15 +27,24 @@ object Library {
   val akkaActor210    = "com.typesafe.akka"     %% "akka-actor"                 % Version.akkaActor210 % "provided" // Apache2
 
   // TESTS
-
-  val specsCore         = "org.specs2"            %% "specs2-core"              % Version.specs % "test"
-  val specsJunit        = "org.specs2"            %% "specs2-junit"             % Version.specs % "test"
-  val specsMock         = "org.specs2"            %% "specs2-mock"              % Version.specs % "test"
-  val specsMatcherExtra = "org.specs2"            %% "specs2-matcher-extra"     % Version.specs % "test"
+  val specs2 = Seq(
+  "org.specs2" %% "specs2-core"          % Version.specs2 % "test",
+  "org.specs2" %% "specs2-junit"         % Version.specs2 % "test",
+  "org.specs2" %% "specs2-mock"          % Version.specs2 % "test",
+  "org.specs2" %% "specs2-matcher-extra" % Version.specs2 % "test"
+  )
+
+  val specs2Scala210 = Seq(
```

Comment:
Not worried about this duplication right now because I think we can remove Scala 2.10 support in another pull request.